ote/src/cljc/ote/time.cljc

```diff
@@ -411,6 +411,12 @@
   [native-date]
   (date-fields->date-time (date-fields native-date)))
 
+(defn native->date
```

Comment:
This function needs some love, removing the commented part and add a short docstring.